contracts/StrategyDAIPoolTogether.sol

```diff
@@ -55,16 +73,35 @@ contract StrategyDAIPoolTogether is BaseStrategyInitializable {
         IERC20(bonus).safeApprove(unirouter, uint256(-1));
     }
 
-    function initializeParent(
+    function _initialize(
```

Comment:
Keep only this and remove initializeThis (not needed anymore), move initializeThis logic here after super invocation, reducing methods.